python/GafferUI/PathListingWidget.py

```diff
@@ -1030,5 +1025,14 @@ def __sectionResized( self, index, oldWidth, newWidth ) :
 
 		# store the difference between the ideal size and what the user would prefer, so
 		# we can apply it again in updateColumnWidths
-		if len( self.__idealColumnWidths ) > index :
-			self.__columnWidthAdjustments[index] = newWidth - self.__idealColumnWidths[index]
+		column = self.__getColumn( index )
+		self.__columnWidthAdjustments[column] = newWidth - self.__idealColumnWidths[column]
```

Comment:
This is what fixes the annoying "jump back to default width when you change tabs" thing in the LightEditor, right? Might be worth a mention in Changes.md.

Reply:
Added in 8d39641.